src/test/scala/com/cognite/sdk/scala/v1/TransformationsTest.scala

```diff
@@ -0,0 +1,208 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+package com.cognite.sdk.scala.v1
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import com.cognite.sdk.scala.common._
+
+import java.time.temporal.ChronoUnit
+import java.time.Instant
+import scala.util.control.NonFatal
+import java.util.UUID
+
+@SuppressWarnings(
+  Array("org.wartremover.warts.NonUnitStatements")
+)
+class TransformationsTest extends CommonDataModelTestHelper with RetryWhile {
+
+  private val client = new GenericClient[IO](
+    "scala-sdk-test",
+    "extractor-bluefield-testing",
+    "https://bluefield.cognitedata.com",
+    authProvider,
+    None,
+    None,
+    None
+  )
+
+  def shortRandomUUID(): String = UUID.randomUUID().toString.substring(0, 8)
+
+  it should "list transformations" in {
+    val res = client.transformations.list(Some(5)).compile.toList.unsafeRunSync()
+    res.size shouldBe 5
+  }
+
+  it should "create and delete transformations" in {
```

Comment:
Wrap this in a `try ... finally delete` as well, and ignore potential failures when deleting. Otherwise, if one of the early assertions fail, the cleanup in `retryWithExpectedResult` won't happen.